pyoverkiz/models.py

```diff
@@ -181,7 +188,19 @@ def __init__(
         self.data_properties = data_properties
         self.type = ProductType(type)
         self.place_oid = place_oid
-        self.protocol = Protocol(self.device_url.split(":")[0])
+
+        match = re.search(DEVICE_URL_RE, device_url)
+
+        self.subsystem_id = None
```

Comment:
Do we need this? You already set it to `None` in the class declaration above?

Reply:
With it, test don't pass:
" AttributeError: 'Device' object has no attribute 'subsystem_id'"

Reply:
> Do we need this? You already set it to `None` in the class declaration above?

As we override the constructor, we bypass the attr.ib declarations.

Reply:
What about the `@define(init=False)` existing on the class definition?  
Could it be related?